### ISSUE TITLE
Current changelog process requires creating PR and then adding changelog file

### ISSUE THREAD

**Issue by skriss** [heptio/velero#1199]
We're asking each non-trivial PR to have a changelog file in the format <pr #>-<username>.  The PR # isn't known until the PR is created (it can be guessed, but this is obviously not ideal).  So in the normal case, this always requires a minimum of two commits/pushes.

It'd be nice to simplify this.  I haven't spent any time yet thinking about possible alternatives yet, but hoping to collect input here.

The reason we're not using issue # is that not every PR has a corresponding issue.  We could require that an issue exist, but this adds overhead to small contributions.

**Comment by dpaez** [comment]
Hi, @skriss I ended up here because of this [tweet](https://twitter.com/carlisia/status/1093579752311410688) :) 

Anyway, I want to share something that we made at GEUT, it's a tool called [chan](https://github.com/geut/chan) that maybe can result helpful to you. It was made to help developers manipulate and maintain changelogs, following ["keep a changelog"](https://keepachangelog.com/en/1.0.0/) style, i.e.: human-readable changelogs. In companion with chan, we have a github app, called [chaas](https://github.com/apps/geut-chaas) which can be used to check if a changelog has changed in the current PR.

We are working on new features for chaas, like: changelog message suggestion and a next version helper (for releases mostly).

If you want to give it a go, feel free to do it. It should adapt very easily to any workflow. Issues/features/ideas are super welcome too. 👍
